zerver/views/realm_playgrounds.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import require_realm_admin
+from zerver.lib.actions import do_add_realm_playground
+from zerver.lib.request import REQ, JsonableError, has_request_variables
+from zerver.lib.response import json_error, json_success
+from zerver.lib.validator import check_string, check_url
+from zerver.models import UserProfile
+
+
+def check_pygments_language(var_name: str, val: object) -> str:
+    s = check_string(var_name, val)
+    valid_pygments_language = re.compile(r"^[ a-zA-Z0-9_+-./#]*$")
```

Comment:
Do we want to do this sort of regular expression check, or instead try to check it's actually in the list of pygments languages that 

I feel like we talked about this before, so probably this is just a case where we should write down the reasoning in a comment here.